src/jsbsim_bridge.cpp

```diff
@@ -169,7 +168,7 @@ bool JSBSimBridge::SetMavlinkInterfaceConfigs(std::unique_ptr<MavlinkInterface>
 
 void JSBSimBridge::Run() {
   // Get Simulation time from JSBSim
-  auto current_time = std::chrono::system_clock::now();
+  auto step_start_time = std::chrono::system_clock::now();
```

Comment:
You're assuming `Run` is executed in a tight loop. This is correct now, and might never change, but just be aware if it does, you'll miss that in the computation.

Reply:
Right, that was the intention of having the `last_step_time` initially